native_client/definitions.mk

```diff
@@ -101,16 +101,34 @@ NODE_PLATFORM_TARGET := --target_arch=arm64 --target_platform=linux
 TOOLCHAIN_LDD_OPTS   := --root $(RASPBIAN)/
 endif # ($(TARGET),rpi3-armv8)
 
+ifeq ($(TARGET),ios-simulator)
```

Comment:
Does that means we have different binaries built for the simulator vs for real hardware ?
Can we run tests on those real hardware binary from the simulator?

Reply:
Hardware is ARM64, simulator is x86_64, so no, can't run tests on real hardware binary.

Reply:
They have no way to run ARM64 simulator? Bad, it means we'll require more dedicated hardware and baby-sitting setup :/

Reply:
I don't think we have the capacity to handle hardware tests for iOS.

Reply:
Do you mean the capacity at iOS simulator level, or our capacity in term of workload? I'm concerned that we have nothing exercizing at all, even basically, the library on ARM64 context, because it's very likely that we will break things somehow :/

Reply:
I mean in terms of the workload. New iPhones + new iOS released every year, Apple is famous for breaking compatibility and forcing developers to change their workflows frequently, plus none of us are iOS developers so everything takes twice as long to setup. I agree it's unfortunate to not have ARM64 test coverage.